tests/packagedcode/data/golang/gomod/milvus/go.mod

```diff
@@ -0,0 +1,255 @@
+module github.com/milvus-io/milvus
```

Comment:
this is quite a large test file, would be nice to truncate this to only have the essential portion required to test the functionality, or find a shorter example. Also add the go.mod files linked in the main issue as tests.